### PR TITLE
Update description,keep consistent with apache.org

### PR DESCRIPTION
**Pull request by chenliang613**
Update description,keep consistent with apache.org

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/545/

File: README.md [hunk 1]
```diff
@@ -19,10 +19,7 @@
 
 <img src="/docs/images/format/CarbonData_logo.png" width="200" height="40">
 
-Apache CarbonData(incubating) is a new big data file format for faster
-interactive query using advanced columnar storage, index, compression
-and encoding techniques to improve computing efficiency, in turn it will 
-help speedup queries an order of magnitude faster over PetaBytes of data. 
+Apache CarbonData(incubating) is a fully indexed columnar and Hadoop native data format for processing heavy analytical workloads and detailed queries on big data.
```
Comment (jackylk) [inline README.md]: I suggest following to describe CarbonData in one setence. 
Apache CarbonData(incubating) is a fully indexed columnar data format for fast analytics on big data platform, e.g. Apache Hadoop, Apache Spark, etc.

Reason: by nature, CarbonData files are indexed and columnar, and the goal is make all analytics faster. 
@gvramana @ravipesala @JihongMA @Vimal-Das @jarray888 what is your opinion?

Comment (JihongMA) [inline README.md]: the suggested revision looks good to me.

Comment (ravipesala) [inline README.md]: @jackylk it looks good, but I think it is better to add about faster detail/point query scenarios also

Comment (chenliang613) [inline README.md]: @ravipesala  in feature list , will mention detail and point query scenarios . so i agree to simplify the description.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/548/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/551/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/553/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/554/

File: README.md [hunk 2]
```diff
@@ -19,10 +19,7 @@
 
 <img src="/docs/images/format/CarbonData_logo.png" width="200" height="40">
 
-Apache CarbonData(incubating) is a new big data file format for faster
-interactive query using advanced columnar storage, index, compression
-and encoding techniques to improve computing efficiency, in turn it will 
-help speedup queries an order of magnitude faster over PetaBytes of data. 
+Apache CarbonData(incubating) is an indexed columnar data format for faster analytics on big data platform like Apache Hadoop, Apache Spark and so on.
```
Comment (jackylk) [inline README.md]: please modify the description in pom.xml also

**Comment by chenliang613** [comment]
Close this PR, create a new PR.

**Pull request closed without merging**